src/go/k8s/tests/e2e/centralized-configuration-tls/00-redpanda-cluster.yaml

```diff
@@ -22,7 +22,6 @@ spec:
     - port: 9644
       tls:
         enabled: true
```

Comment:
just curious: why this change?

Reply:
> There is one regression that will be addressed later if someone would like
to turn off mTLS from internal Admin API new logic does not now how to
handle correct client certification.

:(